### PR TITLE
Adds note on reindexing existing data for semantic_text usage

### PR DESCRIPTION
**Pull request by kosabogi**
## Overview
This update adds a note to the **semantic_text** documentation, clarifying that reindexing is required for existing data to properly generate embeddings for semantic search. 

### Related Issue
[#181](https://github.com/elastic/search-docs-team/issues/181)

### Preview
[Tutorial: semantic search with semantic_text](https://elasticsearch_bk_113590.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/semantic-search-semantic-text.html)

### CODE REVIEW

**Comment by ppf2** [comment]
LGTM from the general reindexing point of view.  It will be helpful to add paragraph that reminds users of web crawlers and connectors that they will have to run a full crawl or full sync after adding semantic_text to crawler or connector mappings, etc.  Thx!

**Comment by kosabogi** [comment]
> LGTM from the general reindexing point of view. It will be helpful to add paragraph that reminds users of web crawlers and connectors that they will have to run a full crawl or full sync after adding semantic_text to crawler or connector mappings, etc. Thx!

Thank you for the review, @ppf2!
I've added a note about full crawls and full syncs in a new commit. 
Let me know if you think any adjustments are needed!

**Review by leemthompo — COMMENTED** [review]
Just a suggestion to streamline the text and keep sentences digestible :)

File: docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc [hunk 1]
```diff
@@ -89,6 +89,17 @@ PUT semantic-embeddings
...
+====
+If you're using web crawlers or connectors to generate indices, you have to <<indices-put-mapping,update>> the existing index mapping
+for the indices generated by web crawlers and connectors to include the `semantic_text` field. Once the mapping is updated,
+you must run a full crawl if you are using a web crawler or a full sync if you are using a connector. This ensures that all
+existing documents are reprocessed and updated with the new semantic embeddings, enabling you to perform semantic search on
+the updated data.
+====
+
+
+
```
[the remark this answers is not in the document]
Comment (ppf2) [reply docs/reference/search/search-your-data/semantic-search-semantic-text.asciidoc:102]: LGTM.  Thanks both!

**Review by szabosteve — APPROVED** [review]

**Pull request merged**